lib/components/Editor/CustomExtensions/Embeds/index.jsx

```diff
@@ -53,28 +53,30 @@ const EmbedOption = ({ isEmbedModalOpen, setIsEmbedModalOpen, editor }) => {
       isOpen={isEmbedModalOpen}
       onClose={handleClose}
     >
-      <Modal.Header>
-        <Typography style="h2">Embed Video</Typography>
-      </Modal.Header>
-      <Modal.Body className="space-y-2">
-        <Input
-          error={error && "Please enter a valid URL"}
-          label="Video URL:"
-          ref={inputRef}
-          size="medium"
-          type="text"
-          value={embedUrl}
-          onChange={handleChange}
-          onKeyDown={handleKeyDown}
-        />
-      </Modal.Body>
-      <Modal.Footer className="space-x-2">
-        <Button
-          disabled={error || isEmpty(embedUrl)}
-          label="Embed"
-          onClick={handleEmbed}
-        />
-      </Modal.Footer>
+      <div className="ne-embed-modal">
+        <Modal.Header>
+          <Typography style="h2">Embed Video</Typography>
+        </Modal.Header>
+        <Modal.Body className="space-y-2">
```

Comment:
```suggestion
        <Modal.Body className="space-y-2 ne-embed-modal">
```

Instead of wrapping the entire `Modal` within a `div`, please add the class within the `Modal.Body`. The code checks for the existence of any element with the class `ne-embed-modal`, we need not add a new div for that.